test/test_nn.py

```diff
@@ -11165,28 +11004,6 @@ def test_conv_cudnn_memory_layout_dominance(self):
         out = conv(input)
         self.assertTrue(out.is_contiguous(memory_format=torch.channels_last))
 
-    def test_conv_double_backward(self):
```

Comment:
Deleted this since i removed the cuda suffix of the migrated test_conv_double_backward_cuda